src/style.css

```diff
@@ -1,3 +1,70 @@
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  --normal-color: black;
+  --hover-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  background-color: white;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
+}
+
+.nav__header {
+  display: flex;
+  align-self: flex-end;
+}
+
+.nav__list {
+  margin: 0;
+  display: flex;
+  list-style: none;
```

Comment:
dont forget to delete padding from list
![opera_HlzQ0qhILf](https://github.com/mate-academy/layout_moyo-header/assets/104684446/ddddcd86-5875-4128-94b0-ed70dbf29c77)